.circleci/config.yml

```diff
@@ -55,6 +55,21 @@ jobs:
               sphinxcontrib-apidoc
 
             conda update --all -y -q
+
+            # install CCL master
```

Comment:
Why did this need to be added now?

Reply:
The ccl feature needed for this is only on master.